man/ctags.1.rst.in

```diff
@@ -1531,6 +1549,8 @@ appear in the general form "key:value". Their presence in the lines of the
 tag file are controlled by the ``--fields`` option. The possible keys and
 the meaning of their values are as follows:
 
+.. Q: this list is very out-of-date. Should we just say "use --list-fields"?
+
```

Comment:
... I wonder what we should do.
I think the list in the man page is well written.
The descriptions in `--list-fields` output are too short to explain the fields.
For many fields, just line line description may be not enough.

There are two ways to go.

1. Introducing `--describe-kind[-<LANG>]=<KIND>`
    This option allows us to embed more descriptions to ctags binaries.
    We can just write "See the output of `--describe-kind` for more details."
    to the man page.

2. Updating the list in the man page

Temporary, my choise is 2.

Reply:
I vote option 3: put them all in the ctags.io online docs, but not the `man` page. (see #1835)